tests/org.eclipse.swt.tests.win32/JUnit Tests/org/eclipse/swt/graphics/TextLayoutWin32Tests.java

```diff
@@ -0,0 +1,39 @@
+package org.eclipse.swt.graphics;
+
+import static org.junit.Assert.assertEquals;
+
+import org.eclipse.swt.internal.DPIUtil;
+import org.eclipse.swt.widgets.Display;
+import org.junit.Before;
+import org.junit.Test;
+
+public class TextLayoutWin32Tests {
+	private Display display;
+	final static String text = "This is a text for testing.";
+
+	@Before
+	public void setUp() {
+		display = Display.getDefault();
+	}
+
+	@Test
+	public void getBoundPublicAPIshouldReturnTheSameValueRegardlessOfZoomLevel() {
+		final TextLayout layout = new TextLayout(display);
+		GCData unscaledData = new GCData();
+		unscaledData.nativeZoom = DPIUtil.getNativeDeviceZoom();
+		GC gc = GC.win32_new(display, unscaledData);
+		layout.draw(gc, 10, 10);
+		Rectangle unscaledBounds = layout.getBounds();
+
+		int scalingFactor = 2;
+		int newZoom = DPIUtil.getNativeDeviceZoom() * scalingFactor;
+		GCData scaledData = new GCData();
+		scaledData.nativeZoom = newZoom;
+		GC scaledGc = GC.win32_new(display, unscaledData);
```

Comment:
Shouldn't it be...
```suggestion
		GC scaledGc = GC.win32_new(display, scaledData);
```
?

Reply:
Yes, corrected